snuba/lw_deletions/strategy.py

```diff
@@ -62,19 +65,25 @@ def submit(self, message: Message[ValuesBatch[KafkaPayload]]) -> None:
 
         try:
             self._execute_delete(conditions)
-        except TooManyOngoingMutationsError:
+        except TooManyOngoingMutationsError as err:
             # backpressure is applied while we wait for the
             # currently ongoing mutations to finish
             self.__metrics.increment("too_many_ongoing_mutations")
+            logger.warning(str(err), exc_info=True)
             raise MessageRejected
+        except QueryException as err:
+            cause = err.__cause__
+            if isinstance(cause, AllocationPolicyViolations):
+                self.__metrics.increment("allocation_policy_violation")
+                raise MessageRejected
 
         self.__next_step.submit(message)
 
     def _get_attribute_info(self) -> AttributionInfo:
         return AttributionInfo(
             app_id=AppID("lw-deletes"),
             tenant_ids={},
-            referrer="lw-deletes",
+            referrer=f"lw-deletes-{self.__storage_name}",
```

Comment:
There are no tenant ids passed from the batch of messages, just the referrer which I've added to be by storage so that we can change the allocation policy per storage

Reply:
Although now that I think about it, the policy is defined on the storage... so this might just be redundant

Reply:
yeah this is redundant, the policies are done by the storage